nengo/builder/operator.py

```diff
@@ -201,6 +201,42 @@ def step():
         return step
 
 
+class SlicedCopy(Operator):
+    """Copy from `a` to `b` with slicing: `b[b_slice] = a[a_slice]`"""
+    def __init__(self, a, b, a_slice=Ellipsis, b_slice=Ellipsis,
+                 kind='set', tag=None):
+        assert kind in ['set', 'inc', 'update']
```

Comment:
Or you can just read the code and know it works ;) 

But fair point. I'll just remove it.